index.js

```diff
@@ -6,22 +6,41 @@ const Move = require('./lib/move')
 const Vec3 = require('vec3').Vec3
 
 const { PlayerState } = require('prismarine-physics')
+const nbt = require('prismarine-nbt')
 
 const THINK_TIMEOUT = 40 // ms
 
 function inject (bot) {
   bot.pathfinder = {}
 
+  function itemIsTool (item) {
+    // tools are named material_toolname eg wooden_pickaxe
+    const itemNameSplit = item.name.split('_')
+    if (itemNameSplit.length === 2) {
+      const toolNames = ['sword', 'axe', 'pickaxe', 'shovel']
+      return toolNames.includes(itemNameSplit[1])
+    }
+    return false
+  }
+
   bot.pathfinder.bestHarvestTool = function (block) {
-    const items = bot.inventory.items()
-    for (const i in block.harvestTools) {
-      const id = parseInt(i, 10)
-      for (const j in items) {
-        const item = items[j]
-        if (item.type === id) return item
+    if (block.name === 'air') return null
```

Comment:
You could use `block.type === 0` id 0 is always air for all mc versions.
But is it really needed ? Does it happen often that bestHarvestTool gets called with an air block as a parameter ?

Reply:
Think this was left over from when I was using the functionality elsewhere, I'll remove it